obs-packaging/linux-container/kata-linux-container.spec-template

```diff
@@ -82,7 +82,7 @@ BuildKernel() {
     find kata-kernel-configs -name "${kernelArch}_kata_kvm_*" -exec cp {} .config \;
 	[ -f .config ] || (echo "ERROR: cannot find the kernel config file for the ${kernelArch} architecture"; exit 1)
 
-    %if 0%{?rhel_version}
+    %if 0%{?rhel_version} || 0%{?suse_version}
```

Comment:
I'm pretty sure this was discussed, but I didn't track it closely.... for my sanity - can you (@jcvenegas and maybe @marcov ) explain to me why we would turn this off - this is a big security fix config, so we should really have it configured imho.

Really, no way around this?

Reply:
This is because some distros has an old gcc version that does not support this issue, last time was added to rhel but not suse I wanted all the distros being building.